README.md

```diff
@@ -1,360 +1,11 @@
-# AppStudio Infrastructure Deployments
+---
+title: StoneSoup Infrastructure Deployments
```

Comment:
Does not render properly -> https://github.com/gbenhaim/infra-deployments/tree/refactor_docs

Reply:
What doesn't render properly? the link you added points to my fork. Can you please provide more information about the issue that you are seeing?

Reply:
The README.md file, when I open your fork from your branch it will show me how it will look like after merge -> https://github.com/gbenhaim/infra-deployments/blob/refactor_docs/README.md - title is in a table

Reply:
fixed